### PR TITLE
Constituency scheme updates

### PR DESCRIPTION
**Pull request by as2388**
- Removes counties which were originally included on ICRAF
 - Corrects a MatchValue

### CODE REVIEW

File: code_schemes/constituency.json [hunk 1]
```diff
@@ -1609,17 +1609,6 @@
                 "kirinyaga central"
             ]
         },
-        {
-            "CodeID": "code-6ff5616f",
-            "DisplayText": "kirinyaga county",
-            "CodeType": "Normal",
-            "NumericValue": 19,
```
Comment (IsaackMwenda) [inline code_schemes/constituency.json:1616]: Should we re-arrange the numeric values?

Comment (as2388) [reply code_schemes/constituency.json:1616]: In my opinion no, as that creates unnecessary risk of confusion.

Comment (IsaackMwenda) [reply code_schemes/constituency.json:1616]: Fair enough.

**Review by IsaackMwenda — APPROVED** [review]
LGTM!

**Review by lukechurch — APPROVED** [review]

**Pull request merged**